packages/style-engine/src/styles/background/index.ts

```diff
@@ -39,7 +39,7 @@ const backgroundImage = {
 };
 
 const backgroundPosition = {
-	name: 'backgroundRepeat',
+	name: 'backgroundPosition',
```

Comment:
I think this might have just been a typo from https://github.com/WordPress/gutenberg/pull/58592